dra-evolution/testdata/pools-two-nodes-dgxa100.yaml

```diff
@@ -0,0 +1,7695 @@
+- apiVersion: resource.k8s.io/v1alpha3
+  kind: ResourcePool
+  metadata:
+    creationTimestamp: null
+    name: nvidia-00-dgxa100
+  spec:
+    attributes:
+    - name: brand
+      string: Nvidia
+    - name: architecture
+      string: Ampere
+    - name: cuda-compute-capability
+      version: 8.0.0
+    - name: driver-version
+      version: 550.54.15
+    - name: cuda-driver-version
+      version: 12.4.0
+    - name: product-name
+      string: Mock NVIDIA A100-SXM4-40GB
+    devices:
+    - attributes:
+      - name: uuid
+        string: GPU-cdd0dd02-b5dd-4ae9-9c5a-c76b19086eb4
+      - bool: true
+        name: mig-capable
+      - bool: false
+        name: mig-enabled
+      name: gpu-0
+      sharedCapacityConsumed:
+      - capacity: 40Gi
+        name: gpu-0-memory
+    - attributes:
+      - name: mig-profile
```

Comment:
We should include in this YAML the requirement that all MIG partitions on a "card" have a "parent UUID" attribute with the same value (for matching).  I think that would highlight the "hidden" nesting level, even if we ultimately choose not to model it.